website/project/model.py

```diff
@@ -2342,6 +2342,12 @@ def remove_contributors(self, contributors, auth=None, log=True, save=False):
                 contributor=contrib, auth=auth, log=False,
             )
             results.append(outcome)
+            if outcome:
+                from website.files.models import OsfStorageFileNode
```

Comment:
To avoid mixing file-related code with the project model, this can be refactored with a signal/listener.